app/blog-content.css

```
@@ -34,9 +34,38 @@
+
+  .blog-content table::-webkit-scrollbar {
+    height: 8px;
+  }
+
+  .blog-content table::-webkit-scrollbar-track {
+    background: transparent;
+  }
+
+  .blog-content table::-webkit-scrollbar-thumb {
+    @apply bg-secondary-400;
+    border-radius: 9999px;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)